unyt/_array_functions.py

```diff
@@ -163,32 +163,62 @@ def histogram(
     a,
     bins=10,
     range=None,
+    density=None,
+    weights=None,
     *args,
     **kwargs,
```

Comment:
there is a backward compatibility issue with this signature (seen in CI): numpy versions `1.19` to `1.23`, which are still supported by `unyt`, also support a `normed` argument that can be either positional of keyword, and inconviently comes *before* `density`. If we want to take `density` into account, we'll need to have a different signature depending on the version of numpy that's available at runtime.
The `NUMPY_VERSION` module constant serves this purpose, see where it's already used for examples !

Reply:
Ah ok I was scratching my head at what was causing the failures, fixed now, hopefully in a sensible way. At least, it will be easy to clean up once numpy versions with `normed` drop out of support.